### DISCUSSION TITLE
Whisper Large-v3 model

### DISCUSSION THREAD

**Discussion by JustAGuyM** [SubtitleEdit/subtitleedit#7870] [Q&A] [answered]
Hi, what version of Subtitle Edit can I download the Large-v3 model of Whisper? I have version 4.0.3, and only Large-v2 is showing up for me

**Comment by niksedk** [comment]
What Whisper engine are you using?
`Const-me` will probably never have v3 is it's not updated anymore.

**Reply by JustAGuyM** [reply]
CPP

**Reply by niksedk** [reply, marked as answer]
The CPP model "Large" is "Large V3" in SE 3.03:

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/2fcb1b52-858f-4e81-9ebf-6acf870253f5)

See https://github.com/SubtitleEdit/subtitleedit/blob/main/src/libse/AudioToText/WhisperCppModel.cs#L158

Calling it"Large V3" would have been easier IMO...

**Reply by JustAGuyM** [reply]
Alright. Thank you!
